test/sanityQueryService.test.js

```diff
@@ -72,7 +76,27 @@ describe('Sanity Queries', function () {
         expect(response).toBeGreaterThan(1000);
     });
 
-    test('fetchByRailContentId', async () => {
+    test('fetchSanity-WithPostProcess', async () => {
+        const id = 380094;
+        const query = `*[railcontent_id == ${id}]{
+        ${getFieldsForContentType('song')}
+          }`
+        const newSlug = 'keysmash1';
+        const newField = 1
+        const postProcess = (result) => {
+            result['new_field'] = newField;
+            result['slug'] = newSlug;
+            return result;
+        };
+        const response = await fetchSanity(query, false, {customPostProcess: postProcess});
+        //console.log(response);
```

Comment:
Just to be consistent
```suggestion
        log(response);
```